subprojects/testfx-spock/testfx-spock.gradle

```diff
@@ -16,83 +16,80 @@
  */
 ext.pomDescription = 'TestFX Spock'
 ext.moduleName = 'org.testfx.spock'
-ext.openjfxVersion = '11'
 
 static def getOSName() {
-    final String osName = System.getProperty("os.name").toLowerCase()
-    if (osName.contains("linux")) {
-        return ("linux")
-    } else if (osName.contains("mac os x") || osName.contains("darwin") || osName.contains("osx")) {
-        return ("mac")
-    } else if (osName.contains("windows")) {
-        return ("win")
-    }
-    return ""
+	final String osName = System.getProperty("os.name").toLowerCase()
```

Comment:
Is it really necessary to replace spaces by tabs? I did not see any `.editorconfig` here and therefore I wonder why this was replaced.

Reply:
Nope. Not necessary. This was an oversight.